blues/data/README.md

```diff
@@ -0,0 +1,3 @@
+Files used for unit testing purposes.
+circle.xml is a force field.
+squareB2 is a simple square model system.
```

Comment:
Again, exactly what? 

Imagine I'm reading this and wanting to know if I can use your file to design a new test to test some other functionality. I have to know more than that it's a square. For example, "squareB2 is a square model system containing N(??) particles arranged in a (size x size) square. It is used for unit testing purposes, particularly to ensure that..." or some such. Remember, the goal here is to make it so I can know what things are here for without having to read your code.